commit 8ed66d3087ef9437bf406569e825d3115db1968d
Author: Josh Salomon <jsalomon@redhat.com>
Date:   Mon Apr 18 11:37:13 2022 +0300

    osd, mon: add read balance score to command `ceph osd pool ls detail`
    for replicated pools.
    
    osd: Added workload balance score to the command
         ceph osd pool ls detail
         (different flavors for console output and json/xml output)
    mon: Added command 'osd rm-primary-temp'
    (developers only) rm was not working
    vstart: Added osd debug messages into mon log with -d flag
            For commands that execute methonds in OSD module
    
    Signed-off-by: Josh Salomon <jsalomon@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index 58c3d9ee4d6..76e9b41f62c 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -235,7 +235,7 @@ PyObject *ActivePyModules::get_python(const std::string &what)
     cluster_state.with_osdmap([&](const OSDMap &osd_map){
       no_gil.acquire_gil();
       if (what == "osd_map") {
-        osd_map.dump(&f);
+        osd_map.dump(&f, g_ceph_context);
       } else if (what == "osd_map_tree") {
         osd_map.print_tree(&f, nullptr);
       } else if (what == "osd_map_crush") {
